pkg/controllers/nodepool/validation/suite_test.go

```diff
@@ -69,7 +69,6 @@ var _ = Describe("Counter", func() {
 		ExpectApplied(ctx, env.Client, nodePool)
 		ExpectObjectReconciled(ctx, env.Client, nodePoolValidationController, nodePool)
 		nodePool = ExpectExists(ctx, env.Client, nodePool)
-		Expect(nodePool.StatusConditions().IsTrue(status.ConditionReady)).To(BeTrue())
```

Comment:
Why do we drop this? Can we hold the same testing line and just make changes to ensure that we can keep this around?